### ISSUE TITLE
Input not updating selected value on mobile

### ISSUE THREAD

**Issue by cool-acid** [lob/react-address-autocomplete#20]
When selecting an address in mobile browsers (and even in the mobile emulation on desktop browsers) the value is not updated with the selected one. Everything else seems to be working correctly. Adding `blurInputOnSelect={false}` to the `Autocomplete` component fixed the issue, but I'm not entirely sure why.

![vh5vng0EZn](https://user-images.githubusercontent.com/602870/177871774-aca72639-a7f0-4069-8385-5f54a97ed187.gif)

After adding `blurInputOnSelect={false}`:
![iv4CdvYgjq](https://user-images.githubusercontent.com/602870/177872155-ebc6aad8-3d8a-4292-9984-b7c91364d95d.gif)

**Comment by Thexumaker** [comment]
Thanks for pointing this out!

So I think I found the root cause:
In /react-address-autocomplete/src/Autocomplete.js on line 269, we are improperly handling input-blurs.

So on mobile devices with react-select, when a user selects an address we not only need to handle a onSelect call but a onInputChange call as well since react-select makes this call by default to remove focus from the input.

We were improperly handling this

```js
    if (action === 'input-blur') {
      // First check is user is controlling input value. If so we use it like for AddressForm
      setInputValue(newInputValue || (selectValue ? selectValue.label : ''))
    }
```

**Comment by Thexumaker** [comment]
@cool-acid The fix should be up on v2.1.3 now. Let me know if the issue is still there but testing it locally it seems like things should be good to go

**Comment by cool-acid** [comment]
@Thexumaker Thank you for your quick response and fix! I updated the library and it works as expected now.
